great-idea-website/css/index.css

```diff
@@ -65,6 +65,12 @@ h1, h2, h3, h4, h5 {
 
 /* Your code starts here! */
 
+.container {
+	width: 880px;
+	margin: 0 auto;
+  }
+
+/* ------   Navigation   ------ */
```

Comment:
These comments that break up the code into manageable chunks is very much appreciated.